jack-sys/src/lib.rs

```diff
@@ -1,8 +1,14 @@
-#![allow(non_camel_case_types, non_upper_case_globals)]
+#![allow(non_camel_case_types, non_upper_case_globals, non_snake_case)]
 
 use dlib::external_library;
 use lazy_static::lazy_static;
 
+#[cfg(windows)]
+pub const JACK_LIB: &'static str = "libjack.dll";
+
+#[cfg(not(windows))]
+pub const JACK_LIB: &'static str = "libjack.so.0\0";
```

Comment:
On macOS, it is `libjack.0.dylib`: https://github.com/jackaudio/jack2/blob/852ba2b8ebc944865e75d8eb5d4550d408772f3b/common/JackWeakAPI.c